packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -338,6 +339,10 @@ export abstract class SfdxCommandletExecutor<T>
     taskViewService.addCommandExecution(execution, cancellationTokenSource);
   }
 
+  public logMetric(logName?: string) {
```

Comment:
New method that will log command metrics if a logName was declared. Will be "automatically" called if `execute` is not overwritten, else we'll have to call it from the new `execute`.